templates/admin/workers/index.html.ep

```diff
@@ -2,19 +2,81 @@
 % title 'Workers';
 
 % content_for 'ready_function' => begin
-    $('#workers').DataTable( { "order": [[2, "desc"],[0, "asc"]] } );
+
+    var table = $('#workers').DataTable( {
```

Comment:
please don't put extensive javascript into the HTML templates. Everything longer than one line should be a function call.